### PR TITLE
Update score for Accessibility Testing (86%)

### PR DESCRIPTION
**Pull request by boazsender**
## Description
Updating the score per the scoring logic in https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/3#issuecomment-1687052499 

## Review Information
You can review this by repeating the math in the above linked issue.

## Changes
This patch changes webapp/components/interop-data.js

## Requirements
No special requirements.

### CODE REVIEW

**Review by cookiecrook — APPROVED** [review]

**Comment by stof** [comment]
@boazsender the CI failure is because you need to do the same change in `webapp/static/interop-data.json`

**Comment by boazsender** [comment]
Thanks for the tip @stof!

File: webapp/components/interop-data.js [hunk 1]
```diff
@@ -318,7 +318,8 @@ export const interopData = {
         'scores_over_time': [
           { 'date': '2023-03-08', 'score': 600 },
           { 'date': '2023-05-13', 'score': 700 },
-          { 'date': '2023-06-27', 'score': 780 }
+          { 'date': '2023-06-27', 'score': 780 },
+          { 'date': '2023-09-05', 'score': 860 }
```
Comment (cookiecrook) [inline webapp/components/interop-data.js:322]:
```suggestion
          { 'date': '2023-09-05', 'score': 860 },
          { 'date': '2023-09-27', 'score': 870 }
```

File: webapp/static/interop-data.json [hunk 2]
```diff
@@ -299,7 +299,8 @@
         "scores_over_time": [
           { "date": "2023-03-08", "score": 600 },
           { "date": "2023-05-13", "score": 700 },
-          { "date": "2023-06-27", "score": 780 }
+          { "date": "2023-06-27", "score": 780 },
+          { "date": "2023-09-05", "score": 860 }
```
Comment (cookiecrook) [inline webapp/static/interop-data.json:303]:
```suggestion
          { "date": "2023-09-05", "score": 860 },
          { "date": "2023-09-27", "score": 870 }
```

Comment (cookiecrook) [reply webapp/static/interop-data.json:303]: One serendipitous benefit to waiting too long to review is that the score has gone up in the meantime!

**Review by cookiecrook — APPROVED** [review]

**Review by stof — APPROVED** [review]

**Review by DanielRyanSmith — APPROVED** [review]

**Comment by DanielRyanSmith** [comment]
I'm going to be handling a deployment today, so I'll get this change in as well. 🙂

**Pull request merged**